nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -41,7 +41,16 @@ in {
       type = types.bool;
       default = true;
       description = ''
-        This option enables support for QEMU/KVM in libvirtd.
+        This option disables support for non-KVM guests in libvirtd (e.g. aarch64 on x86).
+        KVM is available even if this setting is false.
```

Comment:
The naming of this option is misleading, since both states will enable kvm. 
It should be possible without looking at the description to tell what the option does.
I would add the opposite option called `enableEmulation` and add an alias to `nixos/modules/rename.nix` for migration.

Reply:
I would remove it at all